pkg/controller/datavolume/upload-controller_test.go

```diff
@@ -204,6 +204,60 @@ var _ = Describe("All DataVolume Tests", func() {
 		Expect(string(dv.Status.Progress)).To(Equal("N/A"))
 	})
 
+	It("Should adopt a PVC (with annotation)", func() {
+		annotations := map[string]string{"cdi.kubevirt.io/allowClaimAdoption": "true"}
```

Comment:
Maybe use the constant for the annotation name?

Reply:
I actually think it's technically better to not use the shared consts in tests.  Can easily detect when you break an existing API.  We (myself included) are not consistent about this though.

Reply:
Well in other tests in the PR you do use the const, so for consistency at least in the PR I think it makes sense.

Reply:
fair enough